app/models/bus_stops_route.rb

```diff
@@ -9,4 +9,45 @@ class BusStopsRoute < ApplicationRecord
   validates :direction, presence: true
   validates_uniqueness_of :sequence, scope: %i[route direction]
   validates_uniqueness_of :bus_stop, scope: %i[route direction]
+
+  def self.import(stop_hash)
+    stop_hash.each_pair do |route_dir, direction_variants|
+      variants = direction_variants.values
```

Comment:
Might be worth explaining why the keys are there at all, if they're not used for this method.

Reply:
Ok, I did that.